### PR TITLE
refactor(x): refactor code for creating tags [Start #150]

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

### CODE REVIEW

File: migrations/1662648153132_create-unique-constraint-for-title-in-tags-table.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable camelcase */
```
Comment (jocrah) [inline migrations/1662648153132_create-unique-constraint-for-title-in-tags-table.js:1]: @beblicarl please what does this do?

File: migrations/1661202676736_drop-content-and-title-columns-from-posts-tags-table.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 exports.shorthands = undefined
 
 exports.up = (pgm) => {
-    pgm.dropColumns('posts_tags', ['title', 'content'], { ifExists: true })
+	pgm.dropColumns('posts_tags', ['title', 'content'], { ifExists: true })
```
Comment (jocrah) [inline migrations/1661202676736_drop-content-and-title-columns-from-posts-tags-table.js:6]: why this change @beblicarl

Comment (beblicarl) [reply migrations/1661202676736_drop-content-and-title-columns-from-posts-tags-table.js:6]: when i opened this file, there was an eslint error that had to do with the tab spacing, so i just corrected the spacing error

File: migrations/1662648153132_create-unique-constraint-for-title-in-tags-table.js (hunk 1 above)
Comment (beblicarl) [reply migrations/1662648153132_create-unique-constraint-for-title-in-tags-table.js:1]: it does nothing, i should have removed it. I have no camelCase issues

Comment (jocrah) [reply migrations/1662648153132_create-unique-constraint-for-title-in-tags-table.js:1]: okay, kindly do if not being used

**Review by jocrah — APPROVED** [review]
